common/interest_groups/cwp_government_bureaucracy.txt

```diff
@@ -0,0 +1,99 @@
+﻿ig_intelligence_community = {
+	color = {
+		99
+		86
+		71
+	}
+	texture = "gfx/interface/icons/ig_icons/armed_forces.dds"
+	ideologies = {
+		ideology_jingoist
+		ideology_loyalist
+		ideology_patriotic
+	}
+	enable = {
+		always = yes
+	}
+	traits = {
+		ig_trait_patriotic_fervor
+		ig_trait_veteran_consultation
+		ig_trait_materiel_waste
+	}
+	on_enable = {
+	}
+	on_disable = {
+	}
+	on_character_ig_membership = {
+	}
+	pop_potential = {
+		OR = {
```

Comment:
This seems like an odd spread of pop types, it should overlap a bit more with the armed forces type.